src/program/lwaftr/tests/subcommands/monitor_test.py

```diff
@@ -17,37 +16,26 @@
 
 @unittest.skipUnless(SNABB_PCI0, 'NIC not configured')
 @unittest.skipUnless(TAP_IFACE, tap_err_msg)
-class TestMonitor(unittest.TestCase):
+class TestMonitor(BaseTestCase):
 
-    run_cmd_args = (
-        SNABB_CMD, 'lwaftr', 'run',
-        '--name', 'monitor_test',
+    daemon_args = (
+        str(SNABB_CMD), 'lwaftr', 'run',
         '--bench-file', '/dev/null',
-        '--conf', DATA_DIR / 'icmp_on_fail.conf',
+        '--conf', str(DATA_DIR / 'icmp_on_fail.conf'),
         '--on-a-stick', SNABB_PCI0,
         '--mirror', TAP_IFACE,
     )
-
-    monitor_cmd_args = (
-        SNABB_CMD, 'lwaftr', 'monitor',
-        '--name', 'monitor_test',
-        'all',
-    )
```

Comment:
Ideally the monitor test needs more test cases: keyword 'none', set by IPv4 address and setting by PID. I'm fine if those test cases are added in a new PR.